modules/ROOT/pages/provisioning-exoscale.adoc

```diff
@@ -4,7 +4,7 @@ This guide shows how to provision new Fedora CoreOS (FCOS) instances on https://
 
 == Prerequisites
 
-Before provisioning a FCOS instance, you may want to have at hand an Ignition configuration file containing your customizations. If you need one, see xref:producing-ign.adoc[Producing an Ignition File].
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
```

Comment:
I think the "may" wording there refers to the fact that for Exoscale you can provision without an Ignition config presuming you provide SSH keys via the cloud. This is the case also for e.g. AWS, OpenStack, and any other clouds which Afterburn supports fetching SSH keys from. (And actually the AWS quickstart leverages this: https://docs.fedoraproject.org/en-US/fedora-coreos/getting-started/#_booting_on_a_cloud_vm_aws_example).

Though... we don't currently have that wording either for the dedicated OpenStack and AWS pages, so agreed let's just harmonize them for now.

Reply:
Yeah, there were a couple of these extra things sprinkled in – was slightly confused because of the "may" but opted for the harmonisation approach anyway. The examples for AWS and OpenStack are interesting. I can update this pull request and incorporate your feedback and make this better visible. At the same time … why would one install FCOS without a butane config? As a barebones hello-world example?

Reply:
@jlebon please check; I just updated the relevant cloud provider documentations with a link to the Afterburn documentation.
I also restructured the IBM prerequisite section slightly to be more in line with the others.

Reply:
Yup, thanks a lot!

> At the same time … why would one install FCOS without a butane config? As a barebones hello-world example?

It's a simple way to kick the tires on the OS itself before getting involved with Butane/Ignition.

But also, although we strongly recommend using Ignition for provisioning, depending on what you're doing there's nothing inherently wrong with just bringing up the machine and using SSH for configuring.